zebra/zebra_l2.h

```diff
@@ -38,6 +38,13 @@ struct zebra_l2_bridge_vlan {
 	struct zebra_evpn_access_bd *access_bd;
 };
 
+struct zebra_l2_brvlan_mac {
```

Comment:
umm - this commit adds data structures, but ... where's the code that updates them?
frankly, I think you should separate new feature proposals like this out into a separate PR, and offer just the json-specific changes in this PR